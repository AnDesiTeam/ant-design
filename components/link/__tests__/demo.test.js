import demoTest from '../../../tests/shared/demoTest';

demoTest('link', { skip: ['contrast.md', 'link-style.md'] }); // skipping until we get styled-components to work in jest);
