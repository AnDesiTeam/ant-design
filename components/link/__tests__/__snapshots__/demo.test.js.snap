// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/link/demo/basic.md correctly 1`] = `
<a
  class="ant-link"
  href="https://www.taobao.com"
>
  Taobao
</a>
`;
